openedx/adg/admin_dashboard/applications/views.py

```diff
@@ -1,3 +1,60 @@
 """
 Views for `admin_dashboard.applications` app
 """
+from django.shortcuts import render
+from django.views import View
+
+from openedx.adg.admin_dashboard.views import EnsureAdminUserAuthenticatedMixin
+
+from .constants import WEEKLY_SUBMISSIONS_GRAPH_NUM_OF_WEEKS
+from .helpers import (
+    get_new_applications_count_and_percentage_change,
+    get_submitted_applications,
+    get_weekly_application_submissions
+)
+
+
+class ApplicationsDashboardView(EnsureAdminUserAuthenticatedMixin, View):
+    """
+    View to display admin dashboard
+    """
+
+    template_name = 'adg/admin_dashboard/applications/applications_summary.html'
+
+    def get(self, request):
+        """
+        Send statistics related to applications as context
+        """
+        context = self.get_context_data(request.user)
+
```

Comment:
No need for the blank line.

Reply:
There is no convention of when to add and when not to add blank line before return statement according to pep-8. I tend to always add a blank line before return statement for consistency and clarity. I think there's no harm in doing so, rather it increases readability.

Reply:
It does increase readability but when there are multiple statements above return statement. We can show some flexibility. :)